tools/objinfo.ml

```diff
@@ -229,15 +243,23 @@ let print_cms_infos cms =
   printf "Source file: %s\n"
     (match cms.cms_sourcefile with None -> "(none)" | Some f -> f)
 
-let print_general_infos print_name name crc defines iter_cmi iter_cmx =
+let print_general_infos print_name name crc defines arg_descr mbf
+    iter_cmi iter_cmx =
   printf "Name: %a\n" print_name name;
   printf "CRC of implementation: %s\n" (string_of_crc crc);
   printf "Globals defined:\n";
   List.iter print_name_line defines;
+  let () =
```

Comment:
Maybe abstract this logic into a function and call `Option.iter`, similar to the code below.